.github/workflows/deploy-sst.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy SST functions
+on:
+  push:
+    branches:
+      - master
+      - staging
+
+concurrency: deploy-sst-functions-${{ github.ref }}
+env:
+  DIRECTORY: sst
+
+jobs:
+  set_environment:
+    outputs:
+      current_env: ${{ steps.set_environment.outputs.current_env }}
+    runs-on: ubuntu-latest
+    steps:
+      - id: set-environment
+        run: |
+          echo "Running on branch ${{ github.ref }}"
+          if [ "${{ github.ref }}" = "refs/heads/master" ]; then
+            echo "::set-output name=current_env::production"
```

Comment:
this is being deprecated. update with https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/